src/main/java/TestFilter.java

```diff
@@ -0,0 +1,32 @@
+import javax.servlet.*;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+
+@WebFilter("/cars")
+public class TestFilter implements Filter {
+    private static final String SPECIAL_PARAMETER_NAME = "isValid";
+    private static final String ALLOWED_REQUEST_TYPE = "GET";
+    private static final String ERROR_PAGE_PATH = "/main/webapp/WEB-INF/error.jsp";
+
+    @Override
+    public void init(FilterConfig filterConfig) {
```

Comment:
just remove it

Reply:
Filter is interface, we need to reliase all methods.